packages/platform-ios/template/ios/react-native-xcode.sh

```diff
@@ -0,0 +1,6 @@
+set -e
+
```

Comment:
maybe we could put the with-environment.sh here as well?

Reply:
done